src/collection/components/modals/ShareCollectionModal.tsx

```diff
@@ -59,6 +59,10 @@ const ShareCollectionModal: FunctionComponent<ShareCollectionModalProps> = ({
 	const [isCollectionPublic, setIsCollectionPublic] = useState(collection.is_public);
 	const [triggerCollectionPropertyUpdate] = useMutation(UPDATE_COLLECTION);
 
+	const isCollection = () => {
+		return collection.type_id === 3;
```

Comment:
Eventueel enum voor `type_id`voorzien

Reply:
fixed in https://github.com/viaacode/avo2-client/commit/430c85dc9b87545c9332184b0c22dd8cbef01dc9